### ISSUE TITLE
Multiple lines of text on Android

### ISSUE THREAD

**Issue by cvanbeek13** [edsnider/LocalNotificationsPlugin#12]
## Feature Request

Feature description:  On many Android apps, notifications have multiple lines of text.  At first, only the first line is shown (usually with "..." at the end if there is more), but when a user swipes down on the notification, the rest of the message is shown.  Could that be integrated into this plugin?

**Comment by nbaua** [comment]
Yes, This is most required or at least allow word wrap for two-three lines.

**Comment by sorsimu** [comment]
+1 for this feature

**Comment by oicangi05** [comment]
has anyone implement his own solution for this?
